common/persistence/visibility/elasticsearch/client/client_v7.go

```diff
@@ -140,6 +155,56 @@ func (c *clientV7) Search(ctx context.Context, p *SearchParameters) (*elastic.Se
 	return searchService.Do(ctx)
 }
 
+func (c *clientV7) OpenScroll(ctx context.Context, p *SearchParameters, keepAliveInterval string) (*elastic.SearchResult, error) {
+	scrollService := elastic.NewScrollService(c.esClient).
+		Index(p.Index).
+		Query(p.Query).
+		SortBy(p.Sorter...).
+		KeepAlive(keepAliveInterval)
```

Comment:
Yes, it will be automatically closed in 1 min, so client has 1 min to process results and send new request.  Too many scrolls can be a problem. Default value is 500 per ES cluster but it can be changed. This is why PIT is preferable way to continuously iterate over results.